docs/UserGuide.md

```diff
@@ -740,7 +740,15 @@ Format: `ladd INDEX [recurring/END_DATE] date/START_DATE time/TIME_RANGE subject
 You can add multiple pieces of homework to a lesson in TAB.
 </div>
 
-Examples:
+Example: 
+
+Suppose you have a student named Betsy Crowe with index number 7. You have a history lesson with her every Sunday from 11am to 12pm, starting from 20 December 2021, and you want to add it in TAB. The end date is not known, and you are getting paid $35 per hour. For the first lesson, you want to assign assignment 1 as homework. 
+
+1. Type the command `ladd 7 recurring/ date/20 Dec 2021 time/1100-1200 subject/History rates/35 hw/Assignment 1` into the command box.
+2. Press <kbd>Enter</kbd> and you should see that a lesson has been added to the seventh student in the displayed list!
```

Comment:
I think better to mention name rather than index since index is not shown in the picture
```suggestion
2. Press <kbd>Enter</kbd> and you should see that a lesson has been added to Betsy Crowe in the displayed list!
```